sqlgen/db.go

```diff
@@ -188,10 +190,16 @@ func (db *DB) InsertRow(ctx context.Context, row interface{}) (sql.Result, error
 	clause, args := query.ToSQL()
 
 	span, ctx := opentracingkit.MaybeStartSpanFromContext(ctx, "thunder.sqlgen.InsertRow")
-	span.LogFields(log.String("query", clause))
 	defer span.Finish()
+	span.SetTag("query", clause)
+	span.SetTag("queryVariables", args)
 
-	return db.QueryExecer(ctx).Exec(clause, args...)
+	result, err := db.QueryExecer(ctx).Exec(clause, args...)
```

Comment:
Seems like a wrapper function execWithTrace (result, error) would reduce a bunch of code duplication.